web/src/main/java/com/bazaarvoice/emodb/web/scanner/ScanUploadModule.java

```diff
@@ -154,9 +154,11 @@ protected void configure() {
         bind(String.class).annotatedWith(StashRequestTable.class).toInstance(_config.getScanRequestTable());
         bind(Integer.class).annotatedWith(MaxConcurrentScans.class).toInstance(_config.getScanThreadCount());
 
-        bind(new TypeLiteral<Optional<String>>(){}).annotatedWith(Names.named("pendingScanRangeQueueName"))
+        bind(new TypeLiteral<Optional<String>>() {
+        }).annotatedWith(Names.named("pendingScanRangeQueueName"))
```

Comment:
Previous formatting looked more readable for me.

Reply:
agree